modules/nf-core/scimap/spatiallda/tests/main.nf.test

```diff
@@ -0,0 +1,32 @@
+nextflow_process {
+
+    name "Test Process SCIMAP_SPATIALLDA"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "SCIMAP_SPATIALLDA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "scimap"
+    tag "scimap/spatiallda"
+
+    test("phenotyped - csv") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['imaging']['downstream']['phenotyped'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success }
+            )
```

Comment:
Consider adding assertions for output files, preferably through snapshot if they are deterministic, or through existence/header check if not. Check the documentation and feel free to ask for details.

Reply:
We tried but it's non deterministic